stellar/README.md

```diff
@@ -347,3 +347,34 @@ Unpause the contract:
 ```bash
 node stellar/contract.js unpause axelar_gateway
 ```
+
+## Transfer Ownership or Opertorsihp of the Contract
+
+You can transfer ownership or operatorship of the contract.
+
+#### Usage
+
+Check the owner of the contract:
+
+```bash
+node stellar/contract.js owner [contract-name]
```

Comment:
we don't need to detail every cmd, people can use `--help` for most of them. only the important ones can be mentioned